infrastructure/templates/public-api/main.bicep

```diff
@@ -0,0 +1,301 @@
+
+@description('Networking : Included whitelist')
+param storageFirewallRules array = ['0.0.0.0/0']
+
+//Storage Params ------------------------------------------------------------------------------
+@description('Storage Account Name')
+param storageAccountName string = 'core'
+
+@description('Storage : Name of the root fileshare folder name')
+@minLength(3)
+@maxLength(63)
+param fileShareName string = 'data'
+
+@description('Storage : Type of the file share quota')
+param fileShareQuota int = 1
+
+//PostgreSQL Database Params -------------------------------------------------------------------
+@description('Database : PostgreSQL server Name')
+param postgreSQLserverName string
+
+@description('Database : administrator login name')
+@minLength(0)
+param dbAdminName string
+
+@description('Database : administrator password')
+@minLength(8)
+@secure()
+param dbAdminPassword string
+
+@description('Database : Azure Database for PostgreSQL sku name ')
+@allowed([
+  'Standard_B1ms'
+  'Standard_D4ads_v5'
+  'Standard_E4ads_v5'
+])
+param dbSkuName string = 'Standard_B1ms'
+
+@description('Database : Azure Database for PostgreSQL Storage Size in GB')
+param storageSizeGB int = 32
+
+@description('Database : Azure Database for PostgreSQL Autogrow setting')
+param autoGrowStatus string = 'Disabled'
+
+//Container Registry Params ----------------------------------------------------------------
+@minLength(5)
+@maxLength(50)
+@description('Registry : Name of the azure container registry (must be globally unique)')
+param containerRegistryName string
+
+@description('Deploy the Container Registry if you are not using an existing registry')
+param deployRegistry bool
+
+@description('Container App : Specifies the container image to seed to the ACR.')
+param containerSeedImage string
+
+@description('Select if you want to seed the ACR with a base image.')
+param seedRegistry bool
+
+//Container App Params -------------------------------------------------------------------
+@minLength(2)
+@maxLength(32)
+@description('Specifies the name of the container app.')
+param containerAppName string
+
+@description('Specifies the name of the container app environment.')
+param containerAppEnvName string = 'publicapi'
+
+@description('Specifies the name of the log analytics workspace.')
+param containerAppLogAnalyticsName string = 'publicapi'
+
+@description('Container App : Specifies the container image to deploy from the registry <name>:<tag>.')
+param acrHostedImageName string
+
+@description('Specifies the container port.')
+param targetPort int = 80
+
+@description('Select if you want to use a public dummy image to start the container app.')
+param useDummyImage bool
+
+
+//ServiceBus Queue Params -------------------------------------------------------------------
+@description('Name of the Service Bus namespace')
+param namespaceName string = 'processor'
+
+@description('Name of the Queue')
+param queueName string = 'Processorqueue'
```

Comment:
This doesn't follow the normal naming conventions. Would suggest `publicapi-processor`